yocto-update-strategies.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Updates of Yocto-Based Projects: Effective strategies for ongoing maintenance</title>
+    <meta charset="utf-8">
+    <script>
+        classTitle = "Yocto DevDay 2023";
+        courseAuthor = "Maciej Pijanowski & Tomasz Żyjewski";
```

Comment:
I think it's fine if we leave just one name here

Reply:
fixed https://github.com/3mdeb/conferences/pull/7/commits/b30a17b77567d2265207e1a03aab4e6419f10f75